app/graphql/messaging.tsx

```diff
@@ -0,0 +1,19 @@
+import { useApolloClient } from "@apollo/client"
+import { useEffect } from "react"
+import messaging from "@react-native-firebase/messaging"
+import { MainAuthedDocument } from "./generated"
+
+// refetch when we receive an OS notification
+export const MessagingContainer = () => {
+  const client = useApolloClient()
+
+  useEffect(() => {
+    const unsubscribe = messaging().onMessage(async (_remoteMessage) => {
+      client.refetchQueries({ include: [MainAuthedDocument] })
+    })
+
+    return unsubscribe
```

Comment:
Is unsubscribe a function here?

Reply:
that is correct

<img width="732" alt="image" src="https://user-images.githubusercontent.com/11156370/223137711-5327f804-49c8-409d-ab34-f5c5662f468c.png">

why?

Reply:
Sorry I could have phrased the question better. I was just confirming it was a function because I wasn't sure what `.onMessage` returned.